.github/workflows/publish.yml

```diff
@@ -0,0 +1,57 @@
+name: Publishing of Kythera crates
+
+on:
+  workflow_run:
+    workflows: ["Continuous integration"]
+  merge_group:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
```

Comment:
isn't there some redundancy between the `workflow_run` event and the other ones?

Reply:
no, `workflow_run` is the `depends` intra workflows, this is stating that this workflow should only run if `Continuous integration` execute and passes. See [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) for more info

Reply:
thanks joão. your point is really clear.
sorry mine was probably lacking some clarity, questioning the very need for the `merge_group`, `pull_request` and `push` triggers, next to the `workflow_run` condition.

looking forward to your answer.

anyway, this is not blocking to me 👍

Reply:
I totally subscribe to your description of our goal 👍🙏